### PR TITLE
fixed bugs: 877, 800, 772

### PR DESCRIPTION
**Pull request by vasyldzyuba**

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.ts [hunk 1]
```diff
@@ -19,13 +19,19 @@ export class NewsListGalleryViewComponent implements OnInit {
...
   }
 
   private textValidationOfMinCharacters(): string {
     return this.newsText = (this.ecoNewsModel.text.length >= 198) ?
       ((this.ecoNewsModel.text).slice(0, 197) + '...') : (this.ecoNewsModel.text);
   }
 
+  private textValidationOfMinNameCharacters(): string {
+    return this.ecoNewsModel.author.name = (this.ecoNewsModel.author.name.length >= 11) ?
+      ((this.ecoNewsModel.author.name).slice(0, 11) + '...') : (this.ecoNewsModel.author.name);
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.ts:32]: I'm not sure here - is it really important your 'ellipsis'-behavior based on 198/11 symbols?
May you can use native css and based on size of container element?

Comment (vasyldzyuba) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.ts]: okay, I will use native css

File: src/app/component/layout/components/footer/footer.component.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
           <ul class="social-network-links">
             <li>
               <a class="social-links-btn">
-                <img [src]="footerImageList.twitterIcon"
+                <img class="twitter-link-btn" [src]="footerImageList.twitterIcon"
```
Comment (RomanDeveloperAcc) [inline src/app/component/layout/components/footer/footer.component.html:52]: Please, write attributes in separate rows

**Comment by mplesha** [comment]
conflicts

**Comment by Gusvarov** [comment]
> conflicts

Fixed

**Pull request merged**